catalyst/utils/tests/test_config.py

```diff
@@ -0,0 +1,37 @@
+# flake8: noqa
+import argparse
+import pytest
+
+from .. import config
+
+
+def test_parse_config_args():
```

Comment:
and we definitely need to move this thing to travis :)

Reply:
done :)